<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliderDemoWidget</class>
 <widget class="QWidget" name="SliderDemoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>330</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Horizontal</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="value">
         <number>10639</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="horizontalScrollBar">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDial" name="dial">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="wrapping">
         <bool>false</bool>
        </property>
        <property name="notchesVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Vertical</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSlider" name="verticalSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar_3">
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="verticalScrollBar">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
